server/public/utils/fileutils.go

```diff
@@ -111,3 +113,25 @@ func FindDirRelBinary(dir string) (string, bool) {
 	}
 	return found, true
 }
+
+// Valid characters are: alphanumeric, dash, underscore
+var safeFileNameRegex = regexp.MustCompile(`[^\w\-\_]`)
```

Comment:
There are probably other characters that we could permit, but I wanted to play it safe to not introduce another point of failure.